### PR TITLE
[`EmoteLogServerBarEntry`] New Tweak

### PR DESCRIPTION
**Pull request by Sephizor**
Added a new Tweak that tracks the status of the Emote Log in the server bar. This will respond to changes via either the Emotes UI panel, or the slash command /emotelog. Clicking on the entry in the bar will also toggle it.

Emote log off:
image

Emote log on:
image

### CODE REVIEW

File: SimpleTweaksPlugin.cs [hunk 1]
```diff
@@ -45,8 +45,9 @@ public class SimpleTweaksPlugin : IDalamudPlugin {
         public readonly DebugWindow DebugWindow = new DebugWindow();
         public readonly WindowSystem WindowSystem = new WindowSystem("SimpleTweaksPlugin");
         public readonly Changelog ChangelogWindow = new();
-        
-        
+
+        public IDtrBar DtrBar { get; private set; }
```
Comment (MidoriKami) [inline SimpleTweaksPlugin.cs:49]: Don't add this here, add it to the Service Class instead, then access it via Service.DtrBar

File: Tweaks/EmoteLogServerBarEntry.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        _emoteLogBarEntry.OnClick -= DtrEntryClicked;
+        Service.GameConfig.UiConfigChanged -= UiConfigChanged;
+        Plugin.DtrBar.Remove("EmoteLog");
+        _emoteLogBarEntry.Shown = false;
+        base.Disable();
+    }
+
+    public override void Dispose()
+    {
+        base.Dispose();
```
Comment (MidoriKami) [inline Tweaks/EmoteLogServerBarEntry.cs:34]: If you aren't doing anything in dispose, you shouldn't override dispose.

File: Tweaks/EmoteLogServerBarEntry.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+using Dalamud.Game.Gui.Dtr;
+using SimpleTweaksPlugin.TweakSystem;
+
+namespace SimpleTweaksPlugin;
+
+[TweakAuthor("Sythiri")]
+public unsafe class EmoteLogServerBarEntry : Tweak
+{
+    public override string Name => "Emote Log Status in Server Bar";
+    public override string Description => "Show the emote log status in the server bar.";
```
Comment (MidoriKami) [inline Tweaks/EmoteLogServerBarEntry.cs:11]: Name and description can be tweak attributes the same way author is, it is recommended to use these attributes.
Additionally, add `[TweakReleaseVersion(UnreleasedVersion)]`

File: SimpleTweaksPlugin.cs (hunk 1 above)
Comment (SmbAdministrate) [reply SimpleTweaksPlugin.cs]: Ah, thank you. Moved to Service.cs :)

File: Tweaks/EmoteLogServerBarEntry.cs (hunk 2 above)
Comment (SmbAdministrate) [reply Tweaks/EmoteLogServerBarEntry.cs]: Now removed

File: Tweaks/EmoteLogServerBarEntry.cs (hunk 3 above)
Comment (SmbAdministrate) [reply Tweaks/EmoteLogServerBarEntry.cs]: Moved to attributes

**Pull request merged**